_posts/2019-11-20-bazel-1.2.0.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: posts
+title: "Bazel 1.2"
+authors:
+  - philwo
+---
+
+We've just released [Bazel 1.2](https://github.com/bazelbuild/bazel/releases/tag/1.1.0)!
+
+Bazel 1.2 is intended to be backward-compatible with Bazel 1.0, however it contains a single incompatible change that was deemed a bug fix and thus eligible for inclusion.
+
+Please [report](https://github.com/bazelbuild/bazel/issues/new) any update problems.
+
+## Incompatible changes
```

Comment:
Is there a GitHub issue for this?

Reply:
I don't think so. Google-internal reference for this was cl/277098529. I haven't seen any bug on our GitHub issue tracker of someone who ran into this issue.

Reply:
You can at least link to a commit? (or retroactively add a GitHub issue...)

Reply:
Done.

Reply:
Looks like it was rolled back with cl/277929800 (because it was a breaking change)